tomviz/python/Recon_live_tomo.py

```diff
@@ -0,0 +1,69 @@
+import tomviz.operators
+import numpy as np
+import paramiko
+import h5py
+
```

Comment:
We currently don't package either paramiko or h5py, I thought that these dependencies had been removed? Packaging h5py may not be straight forward given that we already link to the hdf5 c library and would have to make sure we package a compatibly version of h5py.